### PR TITLE
Rwahs 297/remove classification from photograph records

### PR DESCRIPTION
**Pull request by kehh**
Hi @mostro20 assigning this to you because ...
somebody's gotta do it ;)

This fixes RWAHS-297 (and makes a small piece of me die).

This PR:
- Updates the ObjectExportView, and returns null for all the classification fields when the ItemType = 'Photograph'
- Updates the sample source and destination data reflecting this change.

The import mapping (`importScripts/rwahsMuseumMapping.xlsx`) gets updated by running the tests, so have updated that (although there is no actual change in the import mapping)

### CODE REVIEW

File: importScripts/sampleRecords/itemData.destination.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       "ca_objects.PhysicalCondition":"Fair \/ Used",
       "ca_objects.Materials":"Oil on pre-grounded Windsor &amp; Newton canvas.",
       "ca_objects.Dimensions.DimensionsValue":"H:44.9cm W:29.6cm",
-      "classificationHierarchy":"Root node for Classification (179); ILLUSTRATIONS (224); Oil (225); portrait (226)"
+      "classificationHierarchy":"Root node for Classification (179); ILLUSTRATIONS (225); Oil (226); portrait (227)"
```
Comment (kehh) [inline importScripts/sampleRecords/itemData.destination.json:14]: These ID's have changed as the `Photograph` classification values are no longer being loaded.

File: importScripts/sampleRecords/itemData.destination.json [hunk 2]
```diff
@@ -161,8 +160,7 @@
       "ca_objects.LatestYear":"1863",
       "ca_objects.SignificanceCriteria":"Historic;Social or Spiritual",
       "ca_objects.StatementOfSignificance":"Historic: \nSocial:",
-      "ca_objects.Dimensions.DimensionsValue":"Large size 28x22 cm",
-      "classificationHierarchy":"Root node for Classification (179); PLACE (238); Building (239); hospitals (240)"
```
Comment (kehh) [inline importScripts/sampleRecords/itemData.destination.json:165]: This is a photograph with no classification.

File: importScripts/sampleRecords/itemData.source.json [hunk 3]
```diff
@@ -1,602 +1,547 @@
...
-  "PhysicalCondition": "0",
-  "Significance_Representativeness": "1",
-  "StatementOfSignificance": "Historic: \r\nRepresentativeness:",
-  "Materials": "35 mm negative",
-  "Classification": "pmg light truck",
-  "PrimaryClass": "COMMUNICATION",
-  "SecondaryClass": "Postal",
-  "TertiaryClass": "pmg light truck"
- }
+[
```
Comment (kehh) [inline importScripts/sampleRecords/itemData.source.json:1]: This file has different whitespace due to an update of my IDE. if you want to see the actual change append `?w=1` to the URL (this hides any whitespace diffs).

File: importScripts/sql/testImports/itemData.sql [hunk 4]
```diff
@@ -71,5 +70,5 @@ SELECT
 FROM
     ObjectsExport
 WHERE
-    PrimaryKey_Object_Table IN(3,6,404,675,679,8,23,25,29,30,47,48,4138,1584,1621,1622,755,1841,3977,4406,3767,4324,4496,5773,679,750,17084,3058,12836,13671,27992,11114,14390,11423,14712,11457,12626,26488,28319,8789,1621,1622)
+    PrimaryKey_Object_Table IN(3,6,404,675,679,8,23,25,47,4138,1584,3977,4406,3767,4324,4496,3058,12836,17084,13671,27992,11114,14390,11423,14712,11457,12626,26488,8789,1621,1622,29,30,48,750,755,1841,5773,28319)
```
Comment (kehh) [inline importScripts/sql/testImports/itemData.sql:73]: These values are the same, just in a different order. Sorry.

File: importScripts/sql/transformations/ObjectsExportView.sql [hunk 5]
```diff
@@ -93,7 +93,9 @@ CREATE OR REPLACE VIEW ObjectsExport AS
         pc.Condition                                              AS PhysicalConditionCode,
         pc.ConditionDescription                                   AS PhysicalConditionDescription,
         # When SecondaryClass is blank then PrimaryClass's value is already in `Classification`
-        CASE WHEN
+        CASE WHEN o.ItemType = 'Photograph'
```
Comment (kehh) [inline importScripts/sql/transformations/ObjectsExportView.sql:96]: The changes in this file are the actual change to the import, basically just returning `NULL` when the record is a Photograph.

File: importScripts/sql/testImports/itemData.sql (hunk 4 above)
Comment (mostro20) [inline importScripts/sql/testImports/itemData.sql:73]: Thanks for letting me know - would have not really know what to look for otherwise :-)

**Comment by mostro20** [comment]
The changes seem logical, not entirely sure why photograph has to be removed, but sounds like something they might request. 

The resulting updates seem logical. The SQL was (unsurprisingly) more sophisticated than what I know, so was enjoyable reading it to get some better ideas of how to structure queries (e.g. had to look up what Coalesce does - but makes sense in this context).

**Comment by kehh** [comment]
Thanks

**Pull request merged**